src/components/heading/heading.scss

```diff
@@ -3,7 +3,7 @@
 @import "./../../style-config/z-index";
 
 .carbon-heading {
-  padding: 24px 0 14px;
+  padding: 24px 0 0;
```

Comment:
Andrew made the styling changes while driving on my machine.
I assume it's because it was adding an extra padding that wasn't needed.
I don't know if he meant to remove it for all the heading components though...